src/Tools/dotnet-dump/Analyzer.cs

```diff
@@ -102,7 +102,7 @@ public Task<int> Analyze(FileInfo dump_path, string[] command)
                 _target.ServiceProvider.AddServiceFactory<SOSHost>(() => new SOSHost(_contextService.Services));
 
                 // Automatically enable symbol server support, default cache and search for symbols in the dump directory
-                _symbolService.AddSymbolServer(msdl: true, symweb: false, symbolServerPath: null, authToken: null, timeoutInMinutes: 0);
```

Comment:
We should probably also have a retry policy here if we are going to add it by default.